docs/source/contributing/developing.adoc

```diff
@@ -89,19 +89,19 @@ For instructions on setting up IDEA, see link:http://hadihariri.com/2015/09/30/s
 [[manage-dependencies]]
 == Dependency Management
 
-{project} uses link:https://github.com/Masterminds/glide[Glide] for dependency management.
+{project} uses link:https://github.com/golang/dep[Dep] for dependency management.
 
-[[install-glide]]
-=== Installing Glide
+[[install-dep]]
+=== Installing Dep
 
-Before you can use Glide you need to download and install it from GitHub:
+Before you can use Dep you need to download and install it from GitHub:
 
 ----
-$ go get github.com/Masterminds/glide
+$ go get github.com/golang/dep
```

Comment:
This need to be fixed as `go get github.com/golang/dep/cmd/dep` as `go get github.com/golang/dep` does not work for me.